docusaurus.config.js

```diff
@@ -173,7 +165,7 @@ const config = {
             ],
           },
         ],
-        copyright: `© ${new Date().getFullYear()} OpenFeature | Documentation Distributed under CC BY 4.0 | All Rights Reserved`,
+        copyright: `© ${new Date().getFullYear()} OpenFeature is a Cloud Native Computing Foundation sandbox project | Documentation Distributed under CC BY 4.0 | All Rights Reserved`,
```

Comment:
Can we put this on two lines now? It's getting pretty long.

Reply:
The text wraps automatically.

Reply:
Okay, let's leave it for now.